wrap/src/lib.rs

```diff
@@ -132,6 +136,20 @@ fn subinvoke(_: &JsValue, args: &[JsValue], ctx: &mut Context<'_>) -> JsResult<J
     result
 }
 
+fn get_implementations(_: &JsValue, args: &[JsValue], ctx: &mut Context<'_>) -> JsResult<JsValue> {
+  let uri = args.get(0).unwrap();
+  let uri: String = uri.as_string().unwrap().to_std_string().unwrap();
+
+  let result = polywrap_wasm_rs::get_implementations::wrap_get_implementations(
+      &uri
+  );
+  let result_json = Value::Array(result.into_iter().map(|r| Value::String(r)).collect());
```

Comment:
msgpack_to_json doesn't work?

Reply:
The result is directly a `Vec<String>`, not a msgpack buffer like with `subinvoke`; that's why I did it like that